### PR TITLE
Multiple Virtualization guide improvements (bsc#1179241)

### PR DESCRIPTION
**Pull request by tbazant**
### Description

This update tries to include multiple unrelated fixes to the virt guides
@aginies please decide whether we should backport these changes to 12 branch as well

### Are there any relevant issues/feature requests?

* bsc#1179241

### To which product versions do the changes apply?

The first column is to be filled by the requester, the second column is to be filled by the doc team after the changes have been backported to the maintenance branches.

| Code 15     | Backport Done
| ----------  | ----------
| [x] 15 SP3  | *(no backport necessary)*
| [x] 15 SP2  | [ ]
| [x] 15 SP1  | [ ]
| [x] 15 SP0  | [ ]

| Code 12     | Backport Done
| ----------  | ----------
| [x] 12 SP5  | [ ]
| [x] 12 SP4  | [ ]
| [x] 12 SP3  | [ ]
| [ ] 12 SP2  | [ ]

### CODE REVIEW

File: xml/vt_scenarios.xml [hunk 1]
```diff
@@ -118,12 +118,12 @@
...
     <listitem>
     <para>
-     Hyper-threading (HT) should be disabled to avoid potential security issues
+     Hyper-threading (HT) should be disabled to avoid potential security issues.
     </para>
    </listitem>
    <listitem>
     <para>
-     VM should not share network / storage / network / Hardware
+     VM should not share network, storage, or network Hardware.
```
Comment (tbazant) [inline xml/vt_scenarios.xml:126]:
```suggestion
     VM should not share network, storage, or network hardware.
```

File: xml/qemu_overview.xml [hunk 2]
```diff
@@ -91,6 +83,11 @@
     <command>qemu-system-x86_64</command>
    </para>
   </listitem>
+  <listitem>
+   <para>
+    <command>qemu-system-arm</command>
+   </para>
+  </listitem>
```
Comment (aginies) [inline xml/qemu_overview.xml:90]: we only provide the matching qemu arch CPU type on SLE. Which means: only qemu-system-x86_64 on x86_64 etc.... so this should be removed.

File: xml/vt_scenarios.xml [hunk 3]
```diff
@@ -103,12 +103,12 @@
...
    <listitem>
     <para>
-     Secure Boot can be used for VMs
+     Secure Boot can be used for VMs.
     </para>
    </listitem>
    <listitem>
     <para>
-     KSM should be avoided
+     KSM should be avoided.
```
Comment (aginies) [inline xml/vt_scenarios.xml:111]: link to "xml id : gloss-vt-acronym-ksm"  would be nice here

**Review by aginies — CHANGES_REQUESTED** [review]

File: xml/qemu_overview.xml (hunk 2 above)
Comment (tbazant) [inline xml/qemu_overview.xml:90]:
```suggestion
<!-- 2021-03-08 tbazant -arch not part of SLE
  <listitem>
   <para>
    <command>qemu-system-arm</command>
   </para>
  </listitem>
  -->
```

File: xml/vt_scenarios.xml (hunk 3 above)
Comment (tbazant) [inline xml/vt_scenarios.xml:111]:
```suggestion
     KSM should be avoided. For more details on KSM, refer to
     <xref linkend="gloss-vt-acronym-ksm"/>.
```

**Review by tbazant — COMMENTED** [review]
Fixed 2 issues

**Pull request merged**